cla_public/templates/base.html

```diff
@@ -84,6 +118,36 @@
 
 {% block javascripts %}
   <script>
+
+    var currentLocation = window.location.href;
+    if (currentLocation.indexOf("n43n3") > 0 || currentLocation.indexOf("n43n14/n97") > 0 || currentLocation.indexOf("n88") > 0 || currentLocation.indexOf("n149") > 0 || currentLocation.indexOf("n86") > 0) {
```

Comment:
Is this matching paths in the domestic abuse category? If so you should be able to read the current category with something like session.stored.category (Add the current category as class on the main content wrapper)

Reply:
I have used the `session.checker.diagnosis_previous_choices` check elsewhere on the page for this reason.  Is that enough?
Should I use `session.stored.category` instead of `session.checker.diagnosis_previous_choices`?